app/src/androidTest/java/ch/sdp/vibester/activity/DownloadActivityTest.kt

```diff
@@ -56,21 +57,21 @@ class DownloadActivityTest {
 
  */
 
-    @Test
-    fun downloadIncorrectSong() {
-        val intent = Intent(ApplicationProvider.getApplicationContext(), DownloadActivity::class.java)
-        val scn: ActivityScenario<DownloadActivity> = ActivityScenario.launch(intent)
-        val songName = "adsfasdgyasdfa"
-
-        onView(withId(R.id.download_songName)).perform(typeText(songName), closeSoftKeyboard())
-        Thread.sleep(100)
-        onView(withId(R.id.download_downloadsong)).perform(click())
-        Thread.sleep(2000)
-
-        onView(withId(R.id.download_songName)).check(matches(withText("")))
-        onView(withId(R.id.download_songName)).check(matches(withHint("Please retry!")))
-
-        val extract = File("storage/emulated/0/Download", "extract_of_$songName")
-        assert(!extract.exists())
-    }
+//    @Test
```

Comment:
maybe explain in a comment why this test is commented

Reply:
done!